crates/nu-utils/src/default_files/doc_env.nu

```diff
@@ -1,9 +1,11 @@
-# Sample Nushell Environment Config File
+# Nushell Environment Config File Documentation
+#
+# version = "0.100.1"
 #
 # Previously, environment variables were typically configured in `env.nu`.
 # In general, most configuration can and should be performed in `config.nu`
 # or one of the autoload directories.
 
 # To pretty-print the in-shell documentation for Nushell's various configuration
 # settings, you can run:
-config nu --sample | nu-highlight | less -R
\ No newline at end of file
+config nu --doc | nu-highlight | less -R
```

Comment:
Nit: it lacks a new line on the end of the file (it is standard on unix)

Reply:
Thanks - That should be picked up by `fmt`, but not for `.nu` files I guess.